tests/lib/eslint/flat-eslint.js

```diff
@@ -3862,6 +3953,66 @@ describe("FlatESLint", () => {
                 assert.strictEqual(results[0].suppressedMessages.length, 0);
             });
 
+            it("should not warn unused 'eslint-disable' comments if 'reportUnusedDisableDirectives = false'.", async () => {
```

Comment:
Not sure what's going on but this test is failing in CI but passing locally for me.